src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/components/summary/GroupInputSummary.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { Typography } from '@material-ui/core';
+
+export interface ISingleInputSumary {
+  formData: any;
+  label: any;
```

Comment:
`string` instead of `any`? Or could label be anything else?

Reply:
Could be a React html component, since we support markdown for labels